docs/releasenotes/release-1.132.md

```diff
@@ -27,6 +27,15 @@
     For opt-in direct controller,
     *   Added `spec.labels` field.
 
+## New features or enhancements:
+
+*   [Storage Bucket](https://cloud.google.com/config-connector/docs/reference/resource-docs/storage/storagebucket)
+    *   Removed immutability constraint on
+        [Storage Bucket](https://cloud.google.com/config-connector/docs/reference/resource-docs/storage/storagebucket)
+        resource's `spec.location` and
+        `spec.customPlacementConfig.dataLocations` fields, allowing them to be
+        updated.
```

Comment:
nit: allowing them to be updated --> This will confuse users since it does not give the context on the rationale. How about "you can follow the guide to [relocate the bucket](https://cloud.google.com/storage/docs/bucket-relocation/relocate-buckets)